src/migrations/1740007960791_AddAgendaItems.ts

```diff
@@ -0,0 +1,92 @@
+import { sql, type Kysely } from 'kysely';
+
+export async function up(db: Kysely<unknown>): Promise<void> {
+  await sql`
+    CREATE TABLE "AgendaItem" (
```

Comment:
> there is now an AgendaItem table, an AgendaItems view, and a ProblemAgendaItems view 🤔
Since the rows in this table are many-to-one with agenda items could we consider naming them after something they are 1-1 with.
Maybe `RawMeetings`?

Reply:
i believe the domain term for what's in the table is a "consideration", but naming the table `Consideration` feels confusing. it could be `AgendaItemConsideration`, which is a bit of a mouthful but that's ok. i definitely don't want to do `Meeting` because a meeting encompasses a full agenda of items.

does the `AgendaItems` view show multiple entries if an agenda item is considered in multiple meetings, or is there just one entry per agenda item reference?

Reply:
Right now the agenda items view is 1-1 with the agenda item number, regardless of how many meetings or motions come out of it.